src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -0,0 +1,161 @@
+"""
+    :copyright: 2022 Inmanta
+    :contact: code@inmanta.com
+    :license: Inmanta EULA
+"""
+import copy
+import datetime
+import json
+import typing
+import uuid
+
+import inmanta.config
+import inmanta.protocol.common
+import inmanta_lsm.const
+import inmanta_lsm.model
+import pytest
+import pytest_inmanta.plugin
+
+# Error message to display when the lsm module is not reachable
+INMANTA_LSM_MODULE_NOT_LOADED = (
+    "The inmanta lsm module is not loaded.\n"
+    "    - If you are using v1 modules: make sure this code is called in a context where the project "
+    "fixture has been executed.\n"
+    "    - If you are using v2 modules: make sure the inmanta-module-lsm is installed in your venv."
+)
+
+
+def reset() -> None:
```

Comment:
I wonder if it would be cleaner to call into the parent reset and re-monkey-patch the client, in order prevent suprises if the reset in lsm is updated.

Reply:
As discussed.